src/components/App.tsx

```diff
@@ -412,6 +419,26 @@ export class App extends React.Component<AppProps, AppState> {
     }
     this.setState({ fiddle });
   }
+  async gist() {
+    this.logLn("Exporting Project ...");
+    const gistURI = await Service.exportProjectToGist(this.project, this.state.fiddle);
+    this.logLn("Project Gist CREATED ");
+    if (gistURI) {
+        this.setState({
+            toast:  gistURI
+        });
+        console.log(`Gist created: ${gistURI}`);
+    } else {
+        console.log("Failed!");
+    }
+  }
+
+  dismissToast(index: number) {
```

Comment:
Shouldn't this be `onDismissToast`? `dismissToast` conveys different message, that I need to call this method to dismiss the toast